.github/actions/build-docs/action.yml

```diff
@@ -0,0 +1,27 @@
+name: 'Build client documentation'
+description: 'Generates client documentation using Javadoc'
+inputs:
+  java-version:
+    description: 'The Java version to use'
+    required: false
+    default: '8'
+  gradle-version:
+    description: 'The Gradle version to use'
+    required: false
+    default: '7.3'
```

Comment:
If the Java version is 8, we should set gradle's default version to 6.8
https://github.com/pinecone-io/pinecone-java-client/blob/main/.github/workflows/pr.yml#L17

Reply:
What's the reason for that, is it a compatibility thing? It doesn't seem to align with the compatibility matrix here so I'm mainly curious why we need to: https://docs.gradle.org/current/userguide/compatibility.html